config_files/training/config_lorem_ipsum.yaml

```diff
@@ -306,7 +306,7 @@ evaluation_subscriber:
   config:
     global_rank: ${settings.cuda_env.global_rank}
     project: modalities_lorem_ipsum
-    mode: ONLINE
+    mode: OFFLINE
     experiment_id: ${settings.experiment_id}
-    directory: "."
+    directory: "wandb_storage"
```

Comment:
For the sake of consistency, we should perhaps apply the same change to all the other config files where it is relevant, e.g. 
* `config_files/training/*.yaml`
* `examples/getting_started/example_config.yaml`
* `examples/library_usage/config_lorem_ipsum.yaml`
* [..]